### ISSUE TITLE
Do you have MA-FCN code

### ISSUE THREAD

**Issue by qubewx** [AlreadyUncle/rl_navigation-pub#1]
Toward Automatic Building Footprint Delineation From Aerial Images Using CNN and Regularization

**Comment by shiqingw** [comment]
No. I'm not the author of that paper. We might have the same name.

**Comment by qubewx** [comment]
Thank you for your reply.
&nbsp;


正方形
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Shiqing ***@***.***&gt;; 
发送时间: 2022年10月30日(星期天) 凌晨2:59
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [shiqingw/rl_navigation-pub] Do you have MA-FCN code (Issue #1)





 
No. I'm not the author of that paper. We might have the same name.
